six/modules/c++/six.sidd/include/six/sidd/Annotations.h

```diff
@@ -35,6 +35,18 @@ struct Annotation
     std::string identifier;
     mem::ScopedCopyablePtr<SFAReferenceSystem> spatialReferenceSystem;
     std::vector<mem::ScopedCloneablePtr<SFAGeometry> > objects;
+
+    bool operator==(const Annotation& rhs) const

```

Comment:
Could you double-check the file, please? I don’t see a constructor. I’ll keep my eyes open for anything else superfluous as I’m making the other changes.

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Sunday, April 03, 2016 9:44 PM
To: ngageoint/six-library <six-library@noreply.github.com>
Subject: Re: [ngageoint/six-library] Equality overloading (#56)


In six/modules/c++/six.sidd/include/six/sidd/Annotations.h<http://cp.mcafee.com/d/2DRPoQ92gOrhovd78VBxdxUTsSOMOe76zBZYS-epophd79EVdETuKYUqerCzBUSyyOyC-qen67Pozm6P9X3BvS50kGxHtendGEqTjBPoXTb8T7fZvAnxMVzHTbFIzxTcInvpVqWtAklrK9YG7DR8OJMddECQPt-jLuZXTLuVKVIDeqR4IMzapdIxO-g8Wh-IEjOAwYKv8x7oBImrffNZM349kITiAGM5oLLcFFKfTdAaJMJZ00Lcfc8IBSm4tbydorcDIel_okelgRKDbwGx4jUzapdIxOVJZx4sr1vF6y0QJM8Wh-IEjOAwYKv8x7oBziWq81EmrQ-kONEwGp-9ZtgB0yq83hg1zXxEFdLI6-G4-Gy65MYSh>:

> @@ -35,6 +35,18 @@ struct Annotation

>      std::string identifier;

>      mem::ScopedCopyablePtr<SFAReferenceSystem> spatialReferenceSystem;

>      std::vector<mem::ScopedCloneablePtr<SFAGeometry> > objects;

> +

> +    bool operator==(const Annotation& rhs) const

Unrelated but again as long as you're in here... Annotation does not need a constructor since all that stuff will default to what it's set to anyway, so go ahead and remove it.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<http://cp.mcafee.com/d/2DRPoQ921J5xYQszCm4S7ztPrb38UsqenTPrUVBxB4QsCzASztWXPxEVKqenzqabaarVEVsovdydorcDIel_ok1iG6JQVsSGxHtendzLsIzss_R-hu73CeLsKCOe7sONtZDBHFShhlKUDOEuvkzaT0QSCrjdTVeZXTLuZXCXCOsVHkiP2cFASO7bV0zF7WOxfai3OVYy4tymNpIY_7TctAvZcKWMWZr_uoogvKYghK-MZqRy7A7m4TNNEVvKMUqeki06hvupjjsvKr8lrxrW01uouohpbII8Wn4qMSpfosH-MEsGxHten1l28DN6kOrp3BPrX28US2_id41FrwhQzZpgDB91Vs-h2eNb6BQQg3gITFYFBzh1kPYjWWxa14Qg6yw37T3hirvodwnpjo1cn>


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.